### ISSUE TITLE
ZigZag Coversion

### ISSUE THREAD

**Issue by Shreyash0713** [aayushi-droid/Edabit-Solutions#311]
Before jumping into Pr first comment for assign.

**Problem statement** : The string "PAYPALISHIRING" is written in a zigzag pattern on a given number of rows like this: (you may want to display this pattern in a fixed font for better legibility)

P   A   H   N
A P L S I I G
Y   I   R
And then read line by line: "PAHNAPLSIIGYIR"

**Probelm Link** :  https://leetcode.com/problems/zigzag-conversion/
**Choose one Cateorgy** - Medium 
**question category** :  String

---
Before submitting a PR please Check some details.

- [x] fileName formet should be in camelcase
    example- returnTheNextNumber.py
- [x]  Choose label for Problem , example- "good first issue", "easy"
- [x]  In file there should be problem statement and link to problem
```cpp
/* Problem-Task : This program will add two numbers
 * Problem Link : https://edabit.com/challenge/SFzHtm63XT6EYNHWY
*/
int main() {
    int bla;
    return 0;
}
```

This repository is vaild for HacktoberFest2020

Read before making PR - [How to contribute on Github](https://www.dataschool.io/how-to-contribute-on-github/)
Good Luck,

**Comment by Shreyash0713** [comment]
Can you please assign it to me.

**Comment by Shreyash0713** [comment]
Resolved, please accept and merge PR #312
